default.nix

```diff
@@ -1,2 +1,65 @@
-{ nixpkgs ? import ./nix/nixpkgs_version.nix }:
-import ./nix/py2.nix { inherit nixpkgs; }
+# Usage:
+#
+#     $ nix-shell --pure --argstr python_version 36
+#
+# to use Python version 3.6.
+
+
+{ nixpkgs ? (import ./nix/nixpkgs_version.nix), python_version ? "27" }:
+let
+  name = "python" + python_version + "Packages";
+  pypkgs = nixpkgs.${name};
+  pysparse = if (python_version == "27") then
+    (import ./nix/pysparse.nix { inherit nixpkgs pypkgs; })
+  else
+    null;
+  gmsh = import ./nix/gmsh.nix { inherit nixpkgs; };
+  skfmm = import ./nix/skfmm.nix { inherit nixpkgs pypkgs; };
+in
+  pypkgs.buildPythonPackage rec {
+    pname = "fipy";
+    version = "3.1.3.dev";
```

Comment:
Why is this hard-wired to build 3.1.3.dev?

Reply:
I removed it altogether. It's not required.